src/polymer-ready.js

```diff
@@ -17,38 +17,51 @@ function showPageAction() {
 };
 
 chrome.runtime.onConnect.addListener(function(port) {
+  var allCustomElements = [];
+
+  function isCustomElement(el) {
+    const isAttr = el.getAttribute('is');
+    return el.localName.includes('-') || isAttr && isAttr.includes('-');
+  }
+
+  function findAllCustomElements(nodes) {
+    nodes.forEach((node) => {
+      if (isCustomElement(node)) { allCustomElements.push(node); }
+      if (node.shadowRoot) { findAllCustomElements(node.shadowRoot.querySelectorAll('*')); }
```

Comment:
@hdelossantos @beaufortfrancois 

Is this safer to ensure Array from the `querySelectorAll`?

```patch
- if (node.shadowRoot) { findAllCustomElements(node.shadowRoot.querySelectorAll('*')); }
+ if (node.shadowRoot) { findAllCustomElements([...node.shadowRoot.querySelectorAll('*')]); }
```